addons/actions/README.md

```diff
@@ -66,14 +76,18 @@ If you wish to process action data before sending them over to the logger, you c
 
 ```js
 import { decorate } from '@storybook/addon-actions';
-
 import Button from './button';
 
+export default {
+  title: 'Button',
+  component: Button,
+};
+
 const firstArg = decorate([args => args.slice(0, 1)]);
 
-storiesOf('Button', module).add('default view', () => (
+export const fromObject = () => (
```

Comment:
what does `fromObject` mean?

Reply:
It's just the name of the story? Nothing significant at all.